.github/workflows/main_workflow.yml

```diff
@@ -216,8 +216,9 @@ jobs:
 
           skaffold build --push=false
           skaffold build -p release --push=false
-          docker build deployments/Dockerfiles/tendermint/ -t valory/open-autonomy-tendermint:0.1.0 --file deployments/Dockerfiles/tendermint/Dockerfile
-          docker build deployments/Dockerfiles/tendermint/ -t valory/open-autonomy-tendermint:1.0.0 --file deployments/Dockerfiles/tendermint/Dockerfile
+          TM_DEPLOYMENT_PATH="deployments/Dockerfiles/tendermint/"
+          TM_IMG_NAME="valory/open-autonomy-tendermint"
+          docker build $TM_DEPLOYMENT_PATH -t $TM_IMG_NAME:0.1.0 -t $TM_IMG_NAME:1.0.0 -t $TM_IMG_NAME:latest
```

Comment:
Thanks! Why are we tagging both `0.1.0` and `1.0.0` @angrybayblade ?

Reply:
We shouldn't be building any image in the CI remember? https://github.com/valory-xyz/open-autonomy/issues/1234

Reply:
I agree, but this would require that we always push to the docker hub when we update our files. Currently, the published image is out of sync.